db/migrate/20200621010723_add_devise_to_user.rb

```diff
@@ -0,0 +1,41 @@
+class AddDeviseToUser < ActiveRecord::Migration[6.0]
+  def change
+    change_table :users do |t|
+      ## Database authenticatable
+      #t.string :email#,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      t.integer  :sign_in_count, default: 0, null: false
+      t.datetime :current_sign_in_at
+      t.datetime :last_sign_in_at
+      t.inet     :current_sign_in_ip
+      t.inet     :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
```

Comment:
Why not confirmable?

Reply:
not yet, we need a mailing server first, in the meantime not having this module will let us develop fast since this module will require that we confirm every user we create